airflow/providers/cncf/kubernetes/operators/kubernetes_pod.py

```diff
@@ -370,11 +370,11 @@ def find_pod(
             label_selector=label_selector,
         ).items
 
-        pod = None
+        pod: Optional[k8s.V1Pod] = None
         num_pods = len(pod_list)
         if num_pods > 1:
```

Comment:
Does it also mean: if more than one pod are found here, we can actually use the name to match & find the Pod we desire and then proceed, rather than raising an exception?